### ISSUE TITLE
Demo site for onboarding new users

### ISSUE THREAD

**Issue by byxorna** [tumblr/collins#553]
It would be neato if there was a demo instance of Collins setup that could illustrate features, and let new users get a feel for Collins. One of the issues I've seen with collins is that the initial onboarding curve is pretty steep. There's a lot of integration glue necessary to graft into your infrastructure, and a lot of usage patterns arent very intuitive. Having a visual illustration of patterns that are common would be awesome (i.e. illustrate how pools are used to group assets, illustrate statuses, illustrate configuration assets acting as VIPs, IPAM, etc etc etc). We could either just throw a stateless container up somewhere (that populates the DB and has a reasonable config on bootup), or setup a real DB that gets reinitialized every day or so.

Thoughts?

@defect @roymarantz @michaeljs1990 @jyundt

**Comment by michaeljs1990** [comment]
This sounds like a nice way to onboard new users although I can't think of any places that offer free container hosts :/ Does tumblr have some EC2 credits chilling around.

Additionally some simple docs with a curl command they can run on a machine they own to populate the demo collins might be helpful to see everything available to you in collins.

**Comment by defect** [comment]
I agree this would be pretty cool, being a bit of a pessimist i also see problems :P What do you think of giving more rigorous and well-documented instructions on spinning up the container and using `scripts/populate.sh`? Maybe we could expand it with more real-looking hosts, some classification assets,  custom attributes, etc.

**Comment by vhp** [comment]
I think it's a great idea.
